### DISCUSSION TITLE
Google Nest Hub - Time format

### DISCUSSION THREAD

**Discussion by TestNamePleaseIgnoreMe** [totaldebug/atomic-calendar-revive#1497] [Support] [answered]
I’m casting on Google Nest Hub and the time format is coming up as 24 hours but on other devices except Google Nest Hub, it’s coming up as 12 hours, I’ve tried nearly everything to change to 12 hours but I’m not having luck. Any ideas?

**Comment by marksie1988** [comment]
The card gets the information from the local device or the HA settings, so it's possible the Google home is set to that and it's getting the setting from there.

**Reply by TestNamePleaseIgnoreMe** [reply]
No luck, Google Hub is set as 12 hrs and HA is set as 12hrs too and I'm still getting 24 hours on the atomic calender only on the Google Nest Hub.

**Reply by sbazzle** [reply, marked as answer]
I had the same problem, but with Home Assistant itself.  Everything was in 24-hour format.  I'm not sure what my initial time format setting was, but in the options for the time format in your User Settings, I chose "Use system locale".  That fixed it for me.  Try bouncing around between the first three options.  For my own example below, they all state the display should be 9:29 AM.  One of those might work for you.

![Capture](https://github.com/totaldebug/atomic-calendar-revive/assets/48576150/c8611394-9439-4722-b166-29189671160a)
